Source/cmake/OptionsWPE.cmake

```diff
@@ -416,6 +416,8 @@ if (USE_LIBBACKTRACE)
     if (NOT LIBBACKTRACE_FOUND)
         message(FATAL_ERROR "libbacktrace is required for USE_LIBBACKTRACE")
     endif ()
+    set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE}${CMAKE_COMPILER_SIZE_OPT_FLAGS} -funwind-tables")
+    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG}${CMAKE_COMPILER_SIZE_OPT_FLAGS} -funwind-tables")
```

Comment:
OK, a few observations:

- OptionsGTK.cmake would need the same change
- You really don't want to add `${CMAKE_COMPILER_SIZE_OPT_FLAGS}` here. That's surely going to override whatever the desired optimization level is with -OS
- All major distros already build with -fasynchronous-unwind-tables. Do you know how adding -funwind-tables will interact with -fasynchronous-unwind-tables? I don't know what will happen, but I fear is -funwind-tables will sabotage -fasynchronous-unwind-tables because you are appending it after the distro-provided flags.

Here are my suggestions:

 - Add the flag in WebKitCompilerFlags.cmake instead of here. It's OK to do so even when not building with libbacktrace.
 - Don't mess with size optimization flags, since that's unrelated
 - Use -fasynchronous-unwind-tables since that's what all distros do; presumably libbacktrace is compatible with that? Also, prepend this flag rather than appending it, since there's no reason to prevent overriding it.

Reply:
i would probably add this where `-fno-exceptions` is added, as this is only needed at all due to that